accessor-alias/sycl-2.2/index.md

```diff
@@ -0,0 +1,686 @@
+# Accessor aliases
+
+## Table of Contents
+
+* [Motivation](#motivation)
+* [Revisions](#revisions)
+* [Summary](#summary)
+* Changes
+  * [Defaulting template parameters](#defaulting-template-parameters)
+  * [Simplifying access modes](#simplifying-access-modes)
+  * [Aliases](#aliases)
+  * [Extending the handler](#extending-the-handler)
+* [Considerations and alternatives](#considerations-and-alternatives)
+
+### TLDR: [Examples](#examples)
+
+## Motivation
+
+Accessors are the cornerstone of SYCL
+as they are used inside and outside kernels to access data
+in a pointer-like manner.
+However, SYCL ecosystem projects have shown that
+they can be a bit awkward to handle due to their verbosity.
+Storing accessors in function objects requires specifying all template parameters,
+and accessors of different access modes are treated as different types,
+even though there isn't much difference in accessing data.
+
+### Existing `local_accessor` alias
+
+SYCL 1.2.1 already has an alias for local memory,
```

Comment:
It might be worth noting here that if this is an alias then you can't use CTAD to further reduce verbosity, perhaps an alternative could be to create a new type, it's a more breaking change, but maybe worth it?

Reply:
There's a section `Class template argument deduction` that discusses this, I've added there that `local_accessor` is also affected.